tools/lwl/README.txt

```diff
@@ -56,7 +56,47 @@ Usage
 =====
 
 No special python modules are needed, it should be possible to run the
-application simply as shown below;
+application simply as shown below:
+
+In the first terminal execute the openocd command to connect to the board.
+Assuming that you already flashed to firmware (nuttx.bin) with the LWL
+console support. For stm32f4discovery board I use this command:
+
+------------------------------------------
+$ sudo openocd -f interface/stlink-v2.cfg -f target/stm32f4x.cfg
```

Comment:
This should just be using `-f board/stm32f4discovery.cfg` which will also remove a lot of the warnings you have here.

Reply:
Good idea, I always use this old way! I will fix it, thank you!